lib/membrane/rtp/vad/is_speaking_estimator.ex

```diff
@@ -1,70 +1,64 @@
+  > **Example**
+  >
+  > If level_threshold is 90, levels are `[80, 90, 100, 90]` and there are 2 levels in a immediate
+  > then immediates would be equal to `[1, 2]` since subunit of `[80, 90]` has 1 item above or equal to
+  > the threshold and subunit `[100, 90]` has 2 such items.
+  >
+  > Same goes for mediums. If medium subunit threshold is 2 and number of subunits is 2
+  > then mediums are equal to `[1]` since the only subunit `[1, 2]` had only one element above or equal to the threshold.
 
   The number of levels the function accepts is equal to the product of subunits per tier.
   This way we compute only one long interval because only one is needed.
   If the number of levels is smaller than the required minimum, the algorithm returns silence.
 
   The most recent interval in each tier serves as a basis for computing an activity score.
   The activity score is a logarithm of a quotient of:
-    - the probability of k active items in n total items under an assumption that a person is speaking (binomial coefficient)
+    - the probability of `k` active items in `n` total items under an assumption that a person is speaking (binomial coefficient)
     - same probability but under an assumption that a person is not speaking (exponential distribution)
 
   The activity score for each tier is then thresholded again.
   A threshold for every tier is given as a module parameter.
   If all activity scores are over the threshold, the algorithm estimates that it is speech. Otherwise silence.
 
-  -----------------------------------------------------------------------------------------------------------
+  -----------------
   Overall the parameters for each tier are:
 
-    - subunits:           number of smaller units in one bigger unit
-    - score_threshold:    number equal or above which the activity score
-                          of the given tier must be to be counted as indicating speech
-    - subunit_threshold:  number equal or above which the number of active subunits must be
-                          for the given tier to be marked as active,
-                          (for immediates it is equal to the threshold given as a estimate_is_speaking/2 argument)
-    - lambda:             parameter for the exponential distribution (element of the activity score computations)
+  | Parameter | Description |
+  |-|-|
+  | `subunits` | number of smaller units in one bigger unit |
+  | `score_threshold`  | number equal or above which the activity score of the given tier must be to be counted as indicating speech |
+  | `subunit_threshold` | number equal or above which the number of active subunits must be for the given tier to be marked as active (for immediates it is equal to the threshold given as a estimate_is_speaking/2 argument) |
+  | `lambda` | parameter for the exponential distribution (element of the activity score computations) |
```

Comment:
This still doesn't show how to configure the params - a code snippet to put in config should handle that